profiles/kentik_snmp/a10_networks/a10-thunder.yml

```diff
@@ -479,6 +479,32 @@ metrics:
       - column:
           name: axVirtualServerPortStatName
           OID: 1.3.6.1.4.1.22610.2.4.3.4.4.1.1.4
+  # A table containing HA information.
+  - MIB: A10-AX-MIB
+    table:
+      name: axHAGroupStatusTable
+      OID: 1.3.6.1.4.1.22610.2.4.3.17.2.2
+    symbols:
+      - name: axHAGroupStatusEntry
+        OID: 1.3.6.1.4.1.22610.2.4.3.17.2.2.1
+      - name: axHAGroupID
+        OID: 1.3.6.1.4.1.22610.2.4.3.17.2.2.1.1
```

Comment:
this is a non-volatile ID field (similar to a name) and should be used as a `metric_tag` value for this table